wiki/Beatmap/Pattern/Mania/Chordstreams/en.md

```diff
@@ -0,0 +1,23 @@
+---
+stub: true
+tags:
+  - chordstreams
+---
+
+# Chordstreams
+
+_For stream types specific to 4-key, refer to [streams](mania/streams)._
+
+**Chordstreams** are a type of [stream](wiki/pattern/mania/streams/en.md#streams) which involve different chord sizes. The stream is usually on 1/4th with chords every 1/1st or 1/2nd snap interval. In keymodes outside of 4-key osu!mania, **chordstream** usually refers to streams involving 3 or more notes within the chords.
```

Comment:
I would maybe clarify here that people call "Light" chordstreams patterns that use chords on 1/1st or maybe 1/2nds whereas "Dense" chordstreams usually use 1/4th for chords too instead than double streams. I personally have never heard double streams before but I am not too big of a 7k guy anyway.